### PR TITLE
Clearer logs around building and deploying go apps

### PR DESCRIPTION
**Pull request by janelletavares**
## Description of change

Fixes https://github.com/meroxa/turbine-project/issues/134

## Type of change

- [ ]  New feature
- [x]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

## How was this tested?

- [ ]  Unit Tests
- [ ]  Tested in staging
- [x]  Tested in minikube

## Demo

Screenshot from 2022-04-06 07-20-20

Screenshot from 2022-04-06 07-41-04
Screenshot from 2022-04-06 07-40-41

### CODE REVIEW

**Comment by janelletavares** [comment]
will rebase this after other things are merged... i think you're gonna like it. worth it wait.

**Comment by raulb** [comment]
@janelletavares I've got this other refactor on `apps deploy` to accommodate languages: https://github.com/meroxa/cli/pull/301. Let me know when this PR is ready so I rebase accordingly... or the other way around.

File: cmd/meroxa/root/apps/deploy.go [hunk 1]
```diff
@@ -328,7 +328,7 @@ func (d *Deploy) getPlatformImage(ctx context.Context, appPath string) (string,
 		return "", err
 	}
 
-	fmt.Print("Getting status for build: ", build.Uuid)
+	fmt.Printf("Getting status for build: %s ", build.Uuid)
```
Comment (janelletavares) [inline cmd/meroxa/root/apps/deploy.go:331]: this just makes it easier to copy the uuid in Linux.. my personal preference

**Review by raulb — APPROVED** [review]
I'll deal with a refactor on https://github.com/meroxa/cli/pull/301.

**Pull request merged**